framework/platform/bpl/linux/bpl_db.cpp

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include "bpl/bpl_db.h"
+
+#include <mapf/common/logger.h>
+
+//////////////////////////////////////////////////////////////////////////////
+/////////////////////////////// Implementation ///////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+namespace beerocks {
+namespace bpl {
+
+bool db_add_entry(const std::string &entry_type, const std::string &entry_name,
+                  const std::unordered_map<std::string, std::string> &values)
+{
+    LOG(TRACE) << __func__ << " - NOT IMPLEMENTED!";
```

Comment:
I think `__func__` is not needed because `LOG` is already writing the function name

Reply:
removed